web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -116,6 +116,19 @@ const BookingModalForm = props => {
           value={formData.isHalfday}
           label="Request a halfday"
         />
+        <Input
+          type="input"
+          className={ isEventBeingUpdated ? null : 'hide'}
+          htmlAttrs={{
+            type: 'input',
+            name: 'updateHolidayReason',
+            placeholder: 'optional',
+          }}
+          value={formData.updateHolidayReason || ''}
+          label="Reason for updating holiday:"
+          labelClass={isEventBeingUpdated ? null : 'hide'}
+          disabled= {!formIsValid ? true : false}
```

Comment:
There's a space between disabled= and the curly bracket plus you're going to get a linting error here for using boolean values in the ternary. Does disabled={!formIsValid} work?

Reply:
This will explain it more :) - https://eslint.org/docs/rules/no-unneeded-ternary